linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_suid_privilege_function/rule.yml

```diff
@@ -58,25 +65,52 @@ warnings:
         Note that these rules can be configured in a
         number of ways while still achieving the desired effect.
 
-ocil_clause: 'the lines are commented out or do not exist'
+ocil_clause: 'the command does not return all lines, or the lines are commented out'
 
 ocil: |-
-      Add or update the following rules in "<tt>/etc/audit/rules.d/audit.rules</tt>":
-
-      {{% if product == "ol8" %}}
-      <pre>-a always,exit -F arch=b32 -S execve -C uid!=euid -k setuid</pre>
-      <pre>-a always,exit -F arch=b64 -S execve -C uid!=euid -k setuid</pre>
-      <pre>-a always,exit -F arch=b32 -S execve -C gid!=egid -k setgid</pre>
-      <pre>-a always,exit -F arch=b64 -S execve -C gid!=egid -k setgid</pre>
-      {{% else %}}
-      <pre>-a always,exit -F arch=b32 -S execve -C uid!=euid -F euid=0 -k setuid</pre>
-      <pre>-a always,exit -F arch=b64 -S execve -C uid!=euid -F euid=0 -k setuid</pre>
-      <pre>-a always,exit -F arch=b32 -S execve -C gid!=egid -F egid=0 -k setgid</pre>
-      <pre>-a always,exit -F arch=b64 -S execve -C gid!=egid -F egid=0 -k setgid</pre>
-      {{% endif %}}
-
-      The audit daemon must be restarted for the changes to take effect.
-
-      <pre># sudo systemctl restart <tt>auditd.service</tt></pre>
+    Verify {{{ full_name }}} audits the execution of privileged functions.
+
+    Check if {{{ full_name }}} is configured to audit the execution of the "execve" system call.
+
+    If audit is using the "auditctl" tool to load the rules, run the following command:
+
+    <pre>$ sudo grep execve /etc/audit/audit.rules</pre>
+
+    If audit is using the "augenrules" tool to load the rules, run the following command:
+
+    <pre>$ sudo grep -r execve /etc/audit/rules.d</pre>
+
+    The output should be the following:
+
+    {{% if product == "ol8" %}}
+    <pre>-a always,exit -F arch=b32 -S execve -C uid!=euid -k setuid</pre>
+    <pre>-a always,exit -F arch=b64 -S execve -C uid!=euid -k setuid</pre>
+    <pre>-a always,exit -F arch=b32 -S execve -C gid!=egid -k setgid</pre>
+    <pre>-a always,exit -F arch=b64 -S execve -C gid!=egid -k setgid</pre>
+    {{% else %}}
+    <pre>-a always,exit -F arch=b32 -S execve -C uid!=euid -F euid=0 -k setuid</pre>
+    <pre>-a always,exit -F arch=b64 -S execve -C uid!=euid -F euid=0 -k setuid</pre>
+    <pre>-a always,exit -F arch=b32 -S execve -C gid!=egid -F egid=0 -k setgid</pre>
+    <pre>-a always,exit -F arch=b64 -S execve -C gid!=egid -F egid=0 -k setgid</pre>
+    {{% endif %}}
+
+fixtext: |-
+    Configure {{{ full_name }}} to audit the execution of the "execve" system call.
+
+    If audit is using the "auditctl" tool to load the rules, then add or update the following file system rules to "/etc/audit/audit.rules", if audit is using the "augenrules" tool to load the rules, then add or update the following file system rules to "/etc/audit/rules.d/audit.rules":
```

Comment:
For the STIG, we need to pick one and use it.  It seems that we generally prefer `auditctl` in the STIG.

Reply:
But the Bash remediation does this for both auditctl and augenrules and we can't have a special bash remediation for a specific profile, build system allows us to have different remediations for different products but not different remediations for different profiles.